RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
  0%|          | 0/14 [00:00<?, ?it/s]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [00:28<03:21, 28.72s/it][A
 25%|██▌       | 2/8 [00:33<01:27, 14.63s/it][A100%|██████████| 8/8 [00:33<00:00,  4.19s/it]
  7%|▋         | 1/14 [00:35<07:47, 35.94s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [01:02<07:14, 62.03s/it][A100%|██████████| 8/8 [01:02<00:00,  7.75s/it]
 14%|█▍        | 2/14 [01:41<10:39, 53.31s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [01:27<10:14, 87.84s/it][A
 50%|█████     | 4/8 [01:30<01:09, 17.30s/it][A
 75%|███████▌  | 6/8 [01:31<00:19,  9.90s/it][A100%|██████████| 8/8 [01:31<00:00, 11.38s/it]
 21%|██▏       | 3/14 [03:16<13:17, 72.51s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [02:07<14:50, 127.26s/it][A100%|██████████| 8/8 [02:07<00:00, 15.91s/it] 
 29%|██▊       | 4/14 [05:27<15:54, 95.44s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [02:33<17:51, 153.07s/it][A100%|██████████| 8/8 [02:33<00:00, 19.13s/it] 
 36%|███▌      | 5/14 [08:07<17:48, 118.72s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [02:57<20:42, 177.44s/it][A
 25%|██▌       | 2/8 [03:02<07:37, 76.18s/it] [A
100%|██████████| 8/8 [03:03<00:00, 12.68s/it][A100%|██████████| 8/8 [03:03<00:00, 22.91s/it]
 43%|████▎     | 6/14 [11:13<18:52, 141.58s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [03:34<25:02, 214.63s/it][A
 25%|██▌       | 2/8 [03:41<09:14, 92.41s/it] [A
 50%|█████     | 4/8 [03:42<02:19, 34.85s/it][A
 62%|██████▎   | 5/8 [03:44<01:13, 24.66s/it][A100%|██████████| 8/8 [03:44<00:00, 28.07s/it]
 50%|█████     | 7/14 [15:07<20:04, 172.01s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [04:16<29:53, 256.28s/it][A
 25%|██▌       | 2/8 [04:19<10:43, 107.22s/it][A100%|██████████| 8/8 [04:19<00:00, 32.39s/it] 
 57%|█████▋    | 8/14 [19:31<20:07, 201.19s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [04:45<33:20, 285.81s/it][A
 25%|██▌       | 2/8 [04:52<12:08, 121.42s/it][A
 38%|███▊      | 3/8 [04:52<05:30, 66.16s/it] [A
 75%|███████▌  | 6/8 [04:58<00:47, 23.72s/it][A100%|██████████| 8/8 [04:58<00:00, 37.33s/it]
 64%|██████▍   | 9/14 [24:36<19:27, 233.51s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [05:24<37:51, 324.46s/it][A
 25%|██▌       | 2/8 [05:32<13:50, 138.49s/it][A100%|██████████| 8/8 [05:32<00:00, 41.60s/it] 
 71%|███████▏  | 10/14 [30:12<17:41, 265.26s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [06:02<42:18, 362.70s/it][A
 50%|█████     | 4/8 [06:05<04:37, 69.45s/it] [A100%|██████████| 8/8 [06:05<00:00, 45.72s/it]
 79%|███████▊  | 11/14 [36:26<14:55, 298.48s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [06:43<47:04, 403.51s/it][A100%|██████████| 8/8 [06:43<00:00, 50.44s/it] 
 86%|████████▌ | 12/14 [43:17<11:05, 332.70s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [07:14<50:38, 434.04s/it][A
 25%|██▌       | 2/8 [07:15<17:58, 179.71s/it][A
 38%|███▊      | 3/8 [07:18<08:14, 98.94s/it] [A
 50%|█████     | 4/8 [07:19<04:00, 60.19s/it][A100%|██████████| 8/8 [07:19<00:00, 54.92s/it]
 93%|█████████▎| 13/14 [50:43<06:07, 367.17s/it]
  0%|          | 0/8 [00:00<?, ?it/s][A
 12%|█▎        | 1/8 [07:39<53:34, 459.21s/it][A
 62%|██████▎   | 5/8 [07:40<03:26, 68.68s/it] [A
 88%|████████▊ | 7/8 [07:42<00:43, 43.03s/it][A100%|██████████| 8/8 [07:42<00:00, 57.78s/it]
100%|██████████| 14/14 [58:35<00:00, 398.62s/it]100%|██████████| 14/14 [58:35<00:00, 251.08s/it]
Exception ignored in: <function Pool.__del__ at 0x2b4d0087b4c0>
Traceback (most recent call last):
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/pool.py", line 268, in __del__
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/queues.py", line 362, in put
AttributeError: 'NoneType' object has no attribute 'dumps'
